packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.html

```diff
@@ -86,6 +87,19 @@ <h3 data-translate="hosting_multisite_git_association_configure"></h3>
                     placeholder="{{ $ctrl.REPOSITORY_PLACEHOLDER }}"
                     required
                 />
+                <input
+                    data-ng-if="$ctrl.isConfiguration"
+                    type="text"
+                    disabled
+                    data-oui-tooltip="{{:: 'hosting_multisite_git_association_reconfigure_warning_url_tooltip' | translate }}"
+                    id="repository_url"
+                    name="repository_url"
+                    class="oui-input mr-2"
+                    data-ng-pattern="$ctrl.REGEX_GIT_REPO"
+                    data-ng-model="$ctrl.model.repositoryUrl"
+                    placeholder="{{ $ctrl.REPOSITORY_PLACEHOLDER }}"
+                    required
+                />
```

Comment:
Why didn't reused the existing input ?

Reply:
ah yes, because did not find a way to use disabled active or not with a boolean

Reply:
Hmmm, merging all in one is not possible since we have a data-oui-tooltip that must appear in one case, and not in the other case.

So keeping like it is now here